hono/test/utils/global-setup.ts

```diff
@@ -0,0 +1,5 @@
+import { start } from './db'
+
+export default async function () {
```

Comment:
What is the purpose of this function?

Reply:
This file acts as a "afterAll" for all test cases. The purpose of this particular file is to stop the database once all test files finish. Essentially this saves us from setting up and tearing down the db on each test case file.